style.css

```diff
@@ -422,34 +201,468 @@ form button:active {
 
 .footer-social ul {
   display: flex;
+  justify-content: center;
   align-items: center;
+  margin: 0 auto;
 }
 
 .footer-social ul li {
   margin: 10px;
 }
 
-.email {
-  margin-top: 24px;
-}
-
-.footer p {
-  color: #42526e;
-  font-weight: 400;
-  font-size: 20px;
-  line-height: 32px;
-  text-align: center;
-  margin-bottom: 330px;
-}
-
-.bar {
-  width: 150px;
-  border-bottom: 10px solid rgb(36, 35, 35);
-  margin: 0 auto;
-  border-radius: 5px;
-  background-color: #fff;
-}
-
-.space {
-  height: 10px;
+/* styles for mobile version */
+
+@media only screen and (max-width: 375px) {
+  body {
```

Comment:
- [ ] Since you're trying to implement desktop view, I'd suggest you get rid of the `max-width: 375px` because the browser understands that you're saying "These styles in this block should only apply to devices with 375px of width or less." As a result, there is chaos after `375px` as shown below: 👇 
![image](https://user-images.githubusercontent.com/46839548/160105458-d4d2407a-af81-452e-9873-777320e4975b.png)